AcloConcurrency/AcloConcurrency/Collection/Cocktails/CocktailDescription/Views/CocktailDescriptionView.swift

```diff
@@ -0,0 +1,131 @@
+import SwiftUI
+
+struct CocktailDescriptionView: View {
+
+    private let image: String
+    private let name: String
+
+    init(image: String, name: String){
+        self.image = image
+        self.name = name
+    }
+
+    var body: some View {
+        VStack(spacing: 24){
+            // Image
+            imageView
+            ScrollView(.vertical, showsIndicators: false){
+                VStack(spacing: 16){
+                    VStack(spacing: 8){
+                        Text("Cocktail")
+                            .frame(maxWidth: .infinity, alignment: .leading)
+                        VStack{
+                            HStack{
+                                Text("Glass")
+                                Spacer()
+                                Text("Flute")
+                            }
+                            .padding(.horizontal)
+                        }
+                        .padding(.vertical)
+                        .frame(maxWidth: .infinity)
+                        .background(Color.gray.opacity(0.7))
+                        .cornerRadius(16)
+                        .overlay(
+                            RoundedRectangle(cornerRadius: 16)
+                                .stroke(Color.gray, lineWidth: 2)
+                        )
+                    }
+                    .padding(.horizontal)
+                    VStack(spacing: 8){
+                        Text("Ingredients")
+                            .frame(maxWidth: .infinity, alignment: .leading)
+                        VStack{
+                            HStack{
+                                Text("Glass")
+                                Spacer()
+                                Text("Flute")
+                            }
+                            .padding(.horizontal)
+                        }
+                        .padding(.vertical)
+                        .frame(maxWidth: .infinity)
+                        .background(Color.gray.opacity(0.7))
+                        .cornerRadius(16)
+                        .overlay(
+                            RoundedRectangle(cornerRadius: 16)
+                                .stroke(Color.gray, lineWidth: 1)
+                        )
+                    }
+                    .padding(.horizontal)
+                }
+            }
+        }
+        .frame(maxWidth: .infinity, alignment: .top)
+        .navigationViewStyle(.stack)
+        .navigationTitle(name)
+        .navigationBarTitleDisplayMode(.inline)
```

Comment:
I don't sure, but I think useless

Reply:
.navigationViewStyle(.stack)
this line?

Reply:
yes

Reply:
and DisplayMode